### DISCUSSION TITLE
Trouble finding where to add parameter to stanza.

### DISCUSSION THREAD

**Discussion by AnansiOmega** [Homebrew/discussions#1011] [Tap maintenance and Homebrew development]
Hello, I've been trying to fix an issue where some apps don't work correctly if they aren't downloaded to the correct location. To fix this issue I need to add a parameter to an existing staza ('app'.) Unfortunately, I'm unable to find where to make these changes. From what I've been able to gather I think the changes that I need to make would be ```Library/Homebrew/cask/artifact/relocated.rb ``` in homebrew core. Any assistance would be greatly appreciated thank you!

**Comment by SMillerDev** [comment]
@Homebrew/cask any ideas?

**Comment by vitorgalvao** [comment]
For context, this is a continuation of https://github.com/Homebrew/homebrew-cask/issues/12858.
